tests/unit/libutil/json-utils.cc

```diff
@@ -55,4 +56,104 @@ TEST(from_json, vectorOfOptionalInts) {
     ASSERT_FALSE(vals.at(1).has_value());
 }
 
+TEST(valueAt, simpleObject) {
+    nlohmann::json simple = nlohmann::json::parse(R"({ "hello": "world" })");
```

Comment:
See https://json.nlohmann.me/api/operator_literal_json/

with 
```c++
using nlohmann::literals::operator ""_json;
```
one can just do
```suggestion
    auto simple = R"({ "hello": "world" })" _json;
```

See `tests/unit/libutil/config.cc` for an example of this